Readme.md

```diff
@@ -70,7 +70,7 @@ For instructions on administrating your Open Library instance and
 build instructions for developers, refer the Developer's
 [Quickstart.md](Quickstart.md) document.
 
-You can find more info digging into this old (and in part outdated) document here: http://code.openlibrary.org/en/latest/
+You can find more info digging into this old (and in part outdated) document here: https://code.openlibrary.org/en/latest/
```

Comment:
the code.openlibrary.org domain is not actually secure. Where is it hosted, and is it something we should still be maintaining?